### PR TITLE
Correct types in tests

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Resolves some type issues that are revealed when strict mode is enabled. (https://github.com/libero/article-store/pull/29)

### CODE REVIEW

File: test/middleware/jsonld.test.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ describe('JSON-LD middleware', (): void => {
   });
 
   it('sets the response JSON-LD body with the dataset', async (): Promise<void> => {
-    const { response } = await makeRequest(null, null, next(null, quads));
+    const { response } = await makeRequest(undefined, undefined, next(undefined, quads));
```
Comment (giorgiosironi) [inline test/middleware/jsonld.test.ts:75]: not that readable in terms of parameters, but didn't change from before

File: test/create-article.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { rdf, schema } from '../src/namespaces';
 
 type Options = {
   id?: QuadSubject;
-  name?: Literal;
+  name?: Literal | null;
```
Comment (giorgiosironi) [inline test/create-article.ts:11]: only example that needs `null` in addition to the `undefined` provided by `?`?

**Review by giorgiosironi — APPROVED** [review]

File: test/create-article.ts (hunk 2 above)
Comment (thewilkybarkid) [reply test/create-article.ts:11]: https://github.com/libero/article-store/blob/a6762f995cdd3c277786b858e789a17282b2790a/test/routes/add-article.test.ts#L55-L56

File: test/middleware/jsonld.test.ts (hunk 1 above)
Comment (thewilkybarkid) [reply test/middleware/jsonld.test.ts:75]: Other option is to use an object, but is a wider refactor.

Comment (giorgiosironi) [reply test/middleware/jsonld.test.ts:75]: (unrelated here I agree)

File: test/create-article.ts (hunk 2 above)
Comment (giorgiosironi) [reply test/create-article.ts:11]: So it's by choice, but it's an error case so wouldn't expect it to mean something in particular like "a title that is currently unknown"

**Pull request merged**